forgotpassword/js/passwordrecovery.js

```diff
@@ -23,7 +22,7 @@ $(document).ready(function()
         }
     });
 
-    function resetFields(){
+    function resetFields() {
```

Comment:
Opening brace should be on a new line

*PHPCodeSnifferBear, severity NORMAL, section `js`.*